api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -104,7 +106,14 @@ message Policy {
   // An optional symbolic expression specifying an access control
   // :ref:`condition <arch_overview_condition>`. The condition is combined
   // with the permissions and the principals as a clause with AND semantics.
-  google.api.expr.v1alpha1.Expr condition = 3;
+  // Only be used when checked_condition is not used.
+  google.api.expr.v1alpha1.Expr condition = 3
+      [(udpa.annotations.field_migrate).oneof_promotion = "expression_specifier"];
+
+  // An optional symbolic expression that has been successfully type checked.
+  // Only be used when condition is not used.
```

Comment:
Are you planning on plumbing this through to the RBAC filter in this PR?

Reply:
I will make a follow-up PR to plumb into RBAC filter.

Reply:
Can you add a `[#not-implemented-hide:]` comment above the field for now until that happens?

Reply:
done